### PR TITLE
Leaderboard final touches

### PR DESCRIPTION
**Pull request by saqibjavaiddev**
In this pull request, I have added some styling to my Leaderboard project. I have added footer styled it, changed styling for buttons, scoreboard and form

### CODE REVIEW

**Review by medaminedev66 — CHANGES_REQUESTED** [review]
Hi @saqibjavaiddev, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that has linters errors. Please ensure that your project doesn't have Linters errors. 

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Comment by saqibjavaiddev** [comment]
Hi, @medaminedev66, Thanks for informing me. Sorry, I forgot about the linter errors. But now fixed

File: src/style.css [hunk 1]
```diff
@@ -79,7 +70,8 @@ h1 {
   padding: 10px;
 }
 
-.add-score form button {
```
Comment (GzimAsani) [inline src/style.css:82]: - [ ] You have used the `display: flex` 3 times. This is bad practice of CSS. Please create a class with a name similar to `d-flex` which is responsible only for displaying flex, and give it to all the necessary elements.

**Review by GzimAsani — CHANGES_REQUESTED** [review]
### Hello @saqibjavaiddev       :wave:   
 Good job so far, you've implemented almost all the requirements asked previously! There are some issues that you still need to work on to go to the next project but you are almost there!
 
 ## Highlights
 * Good work with the styling, it looks good :+1: 
 
 ## Required Changes 

- [x] _Check the comments below :arrow_double_down:_



 ## Optional suggestions
 _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._



Cheers and Happy coding! clapclapclap
 
 Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.
 
 **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
 
 _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by zieeco — APPROVED** [review]
# status: APPROVED :green_circle: :100: 

Hi, @saqibjavaiddev, 


68747470733a2f2f6d656469612e67697068792e636f6d2f6d656469612f78543078657a514755357843444a754350652f67697068792e676966


Good job so far 👍 Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


## Highlights ⚡ 

✅ Descriptive PR 👍 
✅ Nice design 🔥 
✅ Linters are passing 👏 


## Optional suggestions
- N/A

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**